bundler/spec/dependabot/bundler/update_checker/version_resolver_spec.rb

```diff
@@ -184,9 +184,8 @@
 
         let(:dependency_files) { bundler_project_dependency_files("requires_bundler") }
 
-        pending "raises a DependencyFileNotResolvable error" do
-          expect { subject }.
-            to raise_error(Dependabot::DependencyFileNotResolvable)
```

Comment:
This hasn't raised since it was originally committed... it was only marked `pending` because waiting on a follow-on PR to land, but even then wasn't expected to raise per https://github.com/dependabot/dependabot-core/pull/3319/files#r600526250

Reply:
this is a weird test since most dependencies have upgraded to use bundler v2, but since you got it to work I don't mind keeping it around

Reply:
Yeah, the intent of the test makes sense to me--ensure no unexpected exceptions start getting thrown here... but I'm not certain the test setup is correct, ie, is the `guard-bundler ~> 2.2.0` pin actually requesting bundler 1? If so, we should keep this... if not, we should nuke this test case.

Reply:
I don't see anything in there specifically pinning to Bundler 1, I'd say the spec can go.

Reply:
Nuked it.